amida-tech/recruitment-registry
# Feature/RR-844 Integrate File Service

#### What's this PR do?
Replaces the command line zip and aws sdk upload with a call to the new file service (https://github.com/amida-tech/file-repository-service).

#### Related JIRA tickets:
[RR-841](https://jira.amida-tech.com/browse/RR-844)

#### How should this be manually tested?
1. Run an instance of the file repository service with valid aws credentials (https://github.com/amida-tech/file-repository-service/blob/master/README.md)
1. Ensure that the .env file contains a
  - `FILE_SERVICE_BASE_URL` which points to the file service instance domain (e.g. http://localhost:8080), and
  - `RECREG_COHORT_BUCKET` with the bucket that the file service instance is uploading to.
1. Check that tests pass (except those two federated search tests that have been failing for a while
1. Run the backend and frontend
1. Log in as a clinician and create and resend a cohort file
1. Check that the zipped cohort file is uploaded to your s3 and that it unzips with the displayed password.

#### Any background context you want to provide?
#### Screenshots (if appropriate):
